src/main-process/parse-command-line.js

```diff
@@ -177,7 +193,25 @@ module.exports = function parseCommandLine (processArgs) {
   }
 }
 
-function normalizeDriveLetterName (filePath) {
+const readProjectSettingsSync = (filepath, executedFrom) => {
+  try {
+    const readPath = path.isAbsolute(filepath) ? filepath : path.join(executedFrom, filepath)
```

Comment:
any reason to have this function do the path resolution instead of passing in the path already resolved? it seems like you're doing that work at the callsite anyway.

also, i could be wrong but i think the convention in the codebase is to favor function declaration (i.e. use the "function" keyword) for top-level functions like this, unless it's like a single line implicit return deal.